elide-standalone/src/main/java/com/yahoo/elide/standalone/config/ElideStandaloneSettings.java

```diff
@@ -211,17 +211,26 @@ default Integer getMaxRunTimeMinutes() {
      *
      * @return Default: 7
      */
-    default Integer getQueryCleanupDays() {
+    default Integer getAsyncQueryCleanupDays() {
         return 7;
     }
 
+    /**
+     *  Root path specification for the Async Models Swagger endpoint.
+     *
+     * @return Default: /api/v1
+     */
+    default String getAsyncSwaggerPath() {
```

Comment:
Instead of making this the async swagger path - how about a config setting for swagger in general (including non-asyc models)?

Reply:
This should just come from getJsonApiPathSpec()